main.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Communet</title>
+    <link rel="stylesheet" href="style.css"> <!--to connect to the style.css file-->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"/>
+</head>
+<body>
+    <!-- home section start -->
+    <div class="home">
+        <h1>Communet</h1>
```

Comment:
I think it should be like:
```suggestion
        <h1>Commune</h1>
```

Reply:
we talked about changing it to Communet instead of commune but sure ill keep it like before